dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -25,6 +25,17 @@ properties:
         mandatory and if not present it defaults to 1 which is the
         configuration at power-up.
 
+    range:
+      type: int
+      required: false
+      default: 2
+      description: Range in g. Default is power-up configuration.
+      enum:
+        - 16 # 16g (1.952 mg/LSB)
+        -  8 #  8g (0.976 mg/LSB)
+        -  4 #  4g (0.488 mg/LSB)
+        -  2 #  2g (0.244 mg/LSB)
+
```

Comment:
This helpful conversion table will not be present in the bindings index. You may want to move it to the description.